scripts/README.md

```diff
@@ -14,46 +14,87 @@ npx truffle compile
+This Master Safe must have an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account. The following scripts will use this account to propose transactions to the interface. This implies that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
+
+In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. _Otherwise, everyone can steal the funds from your account!_
+
+Setup env variables for the deployment process:
 
 ```
 export PK=<Your Key>
-export GAS_PRICE_GWEI=8
+export GAS_PRICE_GWEI=<look up the suggestion from ethgasstation.info>
 export NETWORK_NAME=<network>
 export MASTER_SAFE=<master safe>
 
 ```
 
-### Confirming multisig orders with Metamask
+### Confirming multisig-transactions on gnosis-safe with Metamask
+
+For the signing process, note that the gas consumption is underestimated. There is currently no proper way to estimate the gas limit. In order to make sure that the gas limit of your transaction is sufficient, increase it to 5.9 million gas. In practice, this limit suffices to deploy a strategy with 20 brackets.
+
+### Deploy the bracket-strategy:
+
+In order to deploy new bracket-trader contracts, place orders on behalf of the newly mined contracts and fund their accounts on the exchange, one only has to run the `complete_liquidity_provision` script.
+It will send one ethereum transaction and send two transactions request to the gnosis-safe interface.
+
+The ethereum transasction will create the bracket-traders. For this transaction the provided private key will be used to pay for the gas.
+The first request of the script will generate orders on behalf of the bracket-traders.
+Please sign this transaction in the gnosis-safe interface and double check that the prices of the orders are as intended - for example in [telegram-mainnet](https://t.me/gnosis_protocol) or [telegram-rinkeby](https://t.me/gnosis_protocol_dev) channels.
+Then continue the script and send out the second request, which will fund the bracket-traders' accounts on the exchange. Notice: One has to wait until the transaction is mined, as otherwise the nonce for the next request will be incorrect.
+Making the requests to the gnosis-interfaces does not cost any gas. However, signing and executing the transactions in the gnosis-safe interface will incur gas costs.
+
+Here is an example script invocation:
+
+```js
+npx truffle exec scripts/complete_liquidity_provision.js --targetToken=1 --stableToken=4 --lowestLimit=150 --hig
+hestLimit=200 --currentPrice=175 --masterSafe=$MASTER_SAFE --investmentTargetToken=10 --investmentStableToken=1000 --fleetSize=20 --network=$NETWORK_NAME
+```
+
+The prices must be specified in terms of 1 target token = x stable tokens.
+
+This example deploys a liquidity strategy with 20 brackets between the prices 150-200 on the pair WETH-USDC.
```

Comment:
I thought the current price was queried from an external price feed so the user wouldn't have to specify an incorrect price...

Reply:
Since external prices feeds aren't always reliable, we want both, user input and external input and then we check that they are matching